### ISSUE TITLE
[Accessibility]: 

### ISSUE THREAD

**Issue by subashreddi6** [sourcegraph/sourcegraph#59199]
### Problem description

We are getting a 404 from a gqlgen dependency on appdash-data
https://sourcegraph.com/sourcegraph/appdash-data
Can this be restored ?

### Expected behavior

200

**Comment by fkling** [comment]
Hi @subashreddi6! The labels you added seem unrelated. If you want to access the `sourcegraph/appdash-data` repository then the correct URL would be https://sourcegraph.com/github.com/sourcegraph/appdash-data . The codehost needs to be part of the repository name.
Please let me know if that fixes your problem.

**Comment by subashreddi6** [comment]
it returns this 
 go install sourcegraph.com/github.com/sourcegraph/appdash-data@latest                                                                                                                                                                                    
go: sourcegraph.com/github.com/sourcegraph/appdash-data@latest: unrecognized import path "sourcegraph.com/github.com/sourcegraph/appdash-data": reading https://sourcegraph.com/github.com/sourcegraph/appdash-data?go-get=1: 404 Not Found

**Comment by burmudar** [comment]
Hi 

> it returns this go install sourcegraph.com/github.com/sourcegraph/appdash-data@latest go: sourcegraph.com/github.com/sourcegraph/appdash-data@latest: unrecognized import path "sourcegraph.com/github.com/sourcegraph/appdash-data": reading https://sourcegraph.com/github.com/sourcegraph/appdash-data?go-get=1: 404 Not Found

Hi @subashreddi6, if you want to install `app-dash` via go then the command should be `go install github.com/sourcegraph/appdash-data@latest`.

If you want to view `sourcegraph/appdash-data` on sourcegraph then the url should be `sourcegraph.com/github.com/sourcegraph/appdash-data`
